lib/commands/publish.js

```diff
@@ -220,7 +220,12 @@ class Publish extends BaseCommand {
       })
     }
     if (manifest.publishConfig) {
-      flatten(manifest.publishConfig, opts)
+      const cliFlags = this.npm.config.data.get('cli').raw
```

Comment:
I'm looking into why `raw` is the only thing that works here.  It shouldn't have to be. `data` should also work fine. Either we'll know why and add a comment or use `data` and fix the code or tests to make it work.

Reply:
Ok it's all good.  I just had to refresh my memory of how the config object works.